packages/client/lib/net/protocol/ethprotocol.ts

```diff
@@ -2,6 +2,7 @@ import { BN, bufferToInt } from 'ethereumjs-util'
 import { Block, BlockHeader, BlockHeaderBuffer } from '@ethereumjs/block'
 import { Chain } from './../../blockchain'
 import { Message, Protocol, ProtocolOptions } from './protocol'
+import { BlockBodyBuffer } from '../../../../block/dist'
```

Comment:
```suggestion
import { BlockBodyBuffer } from '@ethereumjs/block'
```

Reply:
Ah, Visual Studio did this for me, nice catch 👍